### DISCUSSION TITLE
Fun little StarCraft scene!

### DISCUSSION THREAD

**Discussion by alexpineda** [pmndrs/postprocessing#401] [Show and tell]
https://wraiths.imbateam.gg/

Plenty of built-in postprocessing passes here! So thanks again <3

https://github.com/imbateam-gg/wraiths-en-route

**Comment by vanruesc** [comment]
Awesome, thanks for sharing!

I think this could look even better with the new mipmap bloom :grin: 
[L194-L197](https://github.com/imbateam-gg/wraiths-en-route/blob/fa3c8f6ad900b2208305e2d59ffcb1f59cc40d98/src/scene/space-scene.ts#L194-L197):
```ts
const bloomEffect = new BloomEffect({
    blendFunction: BlendFunction.SCREEN,
    mipmapBlur: true,
    intensity: 1.25,
})
```

**Reply by alexpineda** [reply]
![image](https://user-images.githubusercontent.com/586716/187001885-f6d54e3d-26d6-466a-9d00-ccae5fc87d9e.png)
Much more "halo" going on!
![image](https://user-images.githubusercontent.com/586716/187001900-5fa4e5c5-d74d-4756-afd1-a4660d5ad81c.png)
types seem out of date for that feature

anyhow, thank you.
